src/api/features.ts

```diff
@@ -0,0 +1,46 @@
+import axios from 'axios'
+import { BASE_URL } from './common'
+export class Feature {
+    _id: string
```

Comment:
another reason to get the api types into a module 👀

Reply:
Yeah that would be really handy here.